### ISSUE TITLE
Typecheck fails because it can't find definitions for cypress or @testing-library/cypress

### ISSUE THREAD

**Issue by evanstern** [remix-run/indie-stack#239]
### Have you experienced this bug with the latest version of the template?

Yes

### Steps to Reproduce

1. Install the indie stack as described by the README (make sure to run setup and install dependencies)
2. Run `npm run typecheck`

### Expected Behavior

The type check should run and succeed.

### Actual Behavior

The following error is produced:

```
error TS2688: Cannot find type definition file for '@testing-library/cypress'.
  The file is in the program because:
    Entry point of type library '@testing-library/cypress' specified in compilerOptions

  cypress/tsconfig.json:15:34
    15     "types": ["node", "cypress", "@testing-library/cypress"],
                                        ~~~~~~~~~~~~~~~~~~~~~~~~~~
    File is entry point of type library specified here.

error TS2688: Cannot find type definition file for 'cypress'.
  The file is in the program because:
    Entry point of type library 'cypress' specified in compilerOptions

  cypress/tsconfig.json:15:23
    15     "types": ["node", "cypress", "@testing-library/cypress"],
                             ~~~~~~~~~
    File is entry point of type library specified here.


Found 2 errors.
```

**Comment by evanstern** [comment]
As a note:

Downgrading the typescript version in the package.json file from `^5.0.4` to `^4.9.5` fixes this. So I suspect this issue is with a newer major version of TypeScript being used in this project.

**Comment by evanstern** [comment]
> Removing `"cypress", "@testing-library/cypress"` from the list of types in `cypress/tsconfig.json` did the tric for me. It became :
> 
> ```js
> // cypress/tsconfig.json
> // ...
>     "types": [
>       "node"
>     ],
> // ...
> ```

Yeah, I did notice this as well. I'm not sure if this is a good idea or not. The tests do appear to run just fine, though.

**Comment by D-Pagey** [comment]
I'm also experiencing this exact problem setting up a brand new Indie Stack

**Comment by QuentinGibson** [comment]
I'm experience this problem also, downgrading TS fixed my problem like the some of the others.

**Comment by MichaelDeBoey** [comment]
Closed by #241
